ui/src/test/Toggle.test.js

```diff
@@ -0,0 +1,38 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import Toggle from '../Components/Global/Toggle/Toggle'
+import sinon from 'sinon';
+
+describe("Toggle Component", () => {
+
+  it("Should render the Toggle Component", () => {
+    render(<Toggle />)
+
+    const checkbox = screen.getByRole('checkbox')
+    expect(checkbox).toBeInTheDocument();
+  })
+
+  it("Should be active", () => {
+    render(<Toggle active={true}/>)
+
+    const checkbox = screen.getByRole('checkbox')
+    expect(checkbox).toHaveAttribute("checked");
+  })
+  
+  it("Should not be active", () => {
+    render(<Toggle active={false}/>)
+
+    const checkbox = screen.getByRole('checkbox')
+    expect(checkbox).not.toHaveAttribute("checked");
+  })
```

Comment:
Should this be tested? Toggle button works even without this attribute.

I think testing the css/classname to check if the toggle works is a bad idea.